aws/cloudformation/standalone/gen_ai_curriculum/monitoring/alarms/alarmConfigurations.ts

```diff
@@ -76,6 +76,16 @@ const failureMetrics = modelIds.map((modelId, index) => ({
   ),
 }));
 
+
+// Strings to sum the failure (eg, "m7, m8, m9, m10, m11, m12" if 6 models)
```

Comment:
Nice! I think we'd need to update the above metric strings as well (`startMetrics` and `failureMetrics`) right?

Reply:
Ah d'oh, I think I just needed to update the `failureMetrics` slightly. Done!